1229 update slugs and navigation for concepts (#1237)
casper-network/docs: source/docs/casper/concepts/index.md

| [Smart Contracts](./smart-contracts.md) | Learn how to develop smart contracts on Casper | 
| [Dictionaries](./dictionaries.md) | Learn about dictionaries, which are a primary construct for storing and retrieving data on the Casper platform | 
| [Serialization](./serialization-standard.md) | Learn about serializing data types and the Casper Serialization Standard | 
| [Design](./design/index.md) | The high-level design of a Casper network | 
| [Economics](./economics/index.md) | Learn about the Casper on-chain economics | 
| [Glossary](./glossary/index.md) | A compendium of all the terms used in Casper, in alphabetical order |

## Next Steps

Once you understand the basic concepts underpinning Casper, here are some further learning paths you may wish to pursue:

- If you are just getting started and looking for help building your first Casper-based app, we recommend the [beginner tutorials](../resources/beginner/index.md) section.
- If you have already started your Casper education and want to continue with more advanced tutorials, try our [advanced tutorials](../resources/advanced/index.md) section.

> - If you already have experience with or know of the Casper platform and are looking for specific information about a development topic, head over to the [development](../developers/index.md) area.

Reviewer: ```suggestion
- If you have prior experience with the Casper platform and you are looking for specific information about a development topic, head over to the [development](../developers/index.md) area.
```
This sentence just feels a little strange.
